src/App.tsx

```diff
@@ -9,7 +10,7 @@ function App() {
   
   useEffect(() => { 
 
-    fetch(`http://www.omdbapi.com/?apikey=${apiKey}&s=batman`, { 
+    fetch(`http://www.omdbapi.com/?apikey=${apiKey}&s=superman`, { 
```

Comment:
Usually, this kind of operation is extracted on its own file within a directory called `api`.

You can create a method and use it in all the app, but if we are only fetching once here, it's okay.